src/TwigComponent/doc/index.rst

```diff
@@ -122,6 +122,26 @@ and any other components by running:
 
 Take a moment to fist pump - then come back!
 
+You can also use the ``make`` commands to generate the component PHP and twig files :
+
+.. code-block:: terminal
+
+    $ php bin/console make:twig-component Alert
+
+If you want your component to reside in a sub directory like ``src/twig/Components/Alert/`` and make a specific component for a Danger alert you can run:
+
+.. code-block:: terminal
+
+    $ php bin/console make:twig-component Alert\\Danger
+
+Or:
+
+.. code-block:: terminal
+
+    $ php bin/console make:twig-component 'Alert\Danger'
+
+This wil render the php file at ``src\twig\Components\Alert\Danger.php`` and the twig file at ``templates\components\Alert\Danger.html.twig``
+
```

Comment:
```suggestion

.. tip

    If you use the Symfony Maker Bundle (...):

    .. code-block:: terminal

        $ php bin/console make:twig-component Alert
```

Is enough, feel free to complete the first sentence

Reply:
I'd like this block to be later in the doc... maybe with just a link here pointing at this block ? 

I feel it's a bit early to tell people to run "make component" when the full concept, the paths, etc... are not yet explained

Reply:
I'm not sure to understand your point @smnandre, since above this block we already speak about the PHP file and the Twig file: 
<img width="887" alt="image" src="https://github.com/user-attachments/assets/d6b2d537-0e50-49ca-82a0-4dac5d4dd0ec">